src/main.rs

```diff
@@ -101,6 +101,10 @@ struct Args {
     #[clap(short = 'u', long = "update")]
     update: bool,
 
+    /// Disable auto-update, override config file
```

Comment:
I feel like this could be improved: When reading this without knowing about what config options there are, this reads as if auto_update would be on by default. I would much rather have something like

> Disables auto-update, if it was set in the config file.

or

> Ignore the auto-update config setting.

or

> If auto-update is configured, disable it for this run.

My priority here are bottom to top, I like the last one best and the top one the least :)

---

Secondly, the actual key for the setting in config.toml is `auto_update`, with an underscore. This is the first time we mention specific config options in the usage text, so a decision is to be made here. I think it should be "auto_update", what do you think?

Reply:
> If auto-update is configured, disable it for this run

That's also my preferred variant.

> I think it should be "auto_update", what do you think?

No strong opinion, but I think "auto-update" or "auto update" would be fine as well, referring to "the feature" and not "the config variable".

Reply:
My attitude when it comes to comments is rather liberal (e.g. I will use expletives without hesitation if the situation calls for it), so I don't have a strong opinion on this either. But clarity is never a bad thing.